scripts/artifacts-building/containers/build-process.sh

```diff
@@ -140,10 +140,10 @@ openstack-ansible containers/artifact-build-chroot.yml \
 
 # Build the list of roles to build containers for
 role_list=""
-role_list="${role_list} elasticsearch kibana rpc-role-logstash memcached_server"
-role_list="${role_list} os_cinder os_glance os_heat os_horizon os_ironic"
-role_list="${role_list} os_keystone os_neutron os_nova os_swift os_tempest"
-role_list="${role_list} rabbitmq_server repo_server rsyslog_server"
+role_list="${role_list} ansible-elasticsearch memcached_server os_cinder os_glance"
```

Comment:
I agree I think we should skip artifacting the elasticsearch container as it's causing a lot of issues.  How do I go about removing the elasticsearch container for the artifact job?